legacy-gpus/legacy-intel.md

```diff
@@ -1,14 +1,14 @@
 # Legacy Intel
 
-## **Sandy Bridge i3/5/7-2XXX**
-### Highest Supported OS: High Sierra(10.13.6)
-### Initial Supported OS: Snow Leopard(10.6)
+## **Sandy Bridge 2XXX**
+### Highest Supported OS: High Sierra (10.13.6)
+### Initial Supported OS: Snow Leopard (10.6)
 
-Unfortunately, Mojave dropped support for these iGPUs but luckily we can actually get these iGPUs working by using old kexts(though no Metal support so things are a bit iffy). I won't link any of the files myself so do be wary when downloading kexts off the internet. Intial supported introduced with macOS 10.7 and is not supported by the [Intel framebuffer patch](https://www.insanelymac.com/forum/topic/334899-intel-framebuffer-patching-using-whatevergreen/?tab=comments#comment-2626271)
+Unfortunately, Mojave dropped support for these iGPUs, but luckily we can actually get these iGPUs working by using old kexts (though no Metal support so things are a bit iffy). The [OpenCore Legacy Patcher](https://github.com/dortania/OpenCore-Legacy-Patcher/releases) is the best way to add support for these GPUs, however, OCLP is geared more towards real Macs and although it will work, official support is not offered for Hackintoshes. Intial supported introduced with macOS 10.7 and is not supported by the [Intel framebuffer patch](https://www.insanelymac.com/forum/topic/334899-intel-framebuffer-patching-using-whatevergreen/?tab=comments#comment-2626271)
```

Comment:
```suggestion
Unfortunately, Mojave dropped support for these iGPUs, but luckily we can actually get these iGPUs working by using old kexts (though no Metal support so things are a bit iffy). The [OpenCore Legacy Patcher](https://github.com/dortania/OpenCore-Legacy-Patcher/releases) is the best way to add support for these GPUs. However, OCLP is geared towards real Macs and while it will work on Hackintoshes, there is no official support. Intial support was introduced with macOS 10.7 and is not supported by the [Intel framebuffer patching guide](https://www.insanelymac.com/forum/topic/334899-intel-framebuffer-patching-using-whatevergreen/?tab=comments#comment-2626271)
```